src/test/java/pages/HW22/loginPage.java

```diff
@@ -0,0 +1,45 @@
+package pages.HW22;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class loginPage extends basePage {
+
+
+    WebDriver driver;
+    WebDriverWait wait;
```

Comment:
Please, remove code redundancies

Don't repeat yourself! ( In Java, it means don't write the same code repeatedly. )

![image](https://user-images.githubusercontent.com/104761482/213124014-2ae58caf-5b79-46d5-a090-5846053d864b.png)

![image](https://user-images.githubusercontent.com/104761482/213124336-59d1e340-602f-4be7-bf76-c9d67744efc2.png)